var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -73,15 +73,41 @@ def patch(self):
     @property
     def blas_libs(self):
         shared = True if '+shared' in self.spec else False
+        query_parameters = self.spec.last_query.extra_parameters
+        query2libraries = {
+            tuple(): ['libblas'],
+            ('c',): [
+                'libcblas',
+                'libblas',
```

Comment:
since the previous comment is collapsed, for convenience here's the line we discuss: 

either to return both `c` and `fortran` when `:c` is used (current state) or ask users to do `spec[lapack:c,fortran]` to get both at once whereas `:c` returns only `c` part.